missions/2PzD_Template.VR/customization/loadouts.sqf

```diff
@@ -42,8 +42,10 @@
 
 #include "loadoutsVeh\Generic\vehLoadoutGEN.sqf"
```

Comment:
vehLoadoutGEN needs to be first in this file or the General Geardefs never gets called.

Reply:
Do you mean first as in, all the way at the top or at the top of the vehicle loadout files?